database/models/tables.py

```diff
@@ -36,6 +36,24 @@ def add_new_artist(self):
         insert = insert_statement("artist", [], [])
         self.execute_query(insert)
 
+    def get_single_artist_info(self, artist_name):
+        select = select_statement("artist")[:-1]
+        _where = where("artist", "name", "=", f'"{artist_name}"')
+        statement = f"{select} {_where};" 
+        print(f"DEBUG SQL STATEMENT: {statement}")
+        query = self.execute_query(statement).fetchone()
+        return self._select_all_artist_data(query)
+    
+    def get_albums_for_artist(self, artist_name):
+        select = select_statement("album", column=["album.id", "title", "release_date"])[:-1]
+        inner = inner_join("album", "artist", "artist_id", "id")
+        _where = where("artist", "name", "=", f'"{artist_name}"')
+        statement = f"{select} {inner} {_where};"
+        print(f"DEBUG SQL STATEMENT: {statement}")
```

Comment:
and this will want to be come
select album.id, title, release_date from album inner join artist on album.artist_id = artist.id where artist.name = %s


in the same format as above. or you can combine them like i mentioned in my other comment